zeebe/engine/pom.xml

```diff
@@ -116,6 +112,12 @@
     </dependency>
 
     <!-- TEST DEPENDENCIES -->
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>zeebe-scheduler</artifactId>
+      <scope>test</scope>
+    </dependency>
+
```

Comment:
:question: I thought we had removed all dependencies to the scheduler for the engine? :thinking:

Reply:
This is a test scoped dependency because various tests make use of the actor scheduler to run the engine, see for example `StreamProcessorRule`.

Reply:
Oh :( I would have hoped even in the tests we didn't :sweat_smile: